### PR TITLE
docs: add PR and commit guidelines to contributing page

### PR DESCRIPTION
**Pull request by sumeshpremraj**

### CODE REVIEW

File: courses/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,20 +1,30 @@
...
+We use GitHub issues to track bug reports, feature requests, and submitting pull requests.
+
+If you find a bug:
+
+1. Use the GitHub issue search to check whether the bug has already been reported.
 
-*Repository URL*: [https://github.com/linkedin/school-of-sre](https://github.com/linkedin/school-of-sre)
+1. If the issue has been fixed, try to reproduce the issue using the latest master branch of the repository.
+
+1. If the issue still reproduces or has not yet been reported, try to isolate the problem before opening an issue.
```
Comment (sanketplus) [inline courses/CONTRIBUTING.md:13]: numbering is incorrect

File: courses/CONTRIBUTING.md [hunk 2]
```diff
@@ -24,7 +34,50 @@ mkdocs build
 mkdocs serve
 ```
 
-### Opening a PR
-Follow the [GitHub PR workflow](https://guides.github.com/introduction/flow/) for your contributions.
+### Submitting a Pull Request (PR)
+
+Before you submit your Pull Request (PR), consider the following guidelines:
+
+* Search this repo for an open or closed PR that relates to your submission. You don't want to duplicate effort.
```
Comment (sanketplus) [inline courses/CONTRIBUTING.md:41]: duplicate point?

**Review by sanketplus — CHANGES_REQUESTED** [review]

File: courses/CONTRIBUTING.md (hunk 1 above)
Comment (sumeshpremraj) [reply courses/CONTRIBUTING.md:13]: This is correct markdown syntax, see https://www.markdownguide.org/basic-syntax/#ordered-lists

File: courses/CONTRIBUTING.md (hunk 2 above)
Comment (sumeshpremraj) [reply courses/CONTRIBUTING.md]: Removed.

**Pull request closed without merging**